ios/Demo-iOS/Sources/GutenbergApp.swift

```diff
@@ -12,12 +37,19 @@ struct GutenbergApp: App {
 
     var body: some Scene {
         WindowGroup {
-            NavigationStack {
+            NavigationStack(path: $navigation.path) {
                 AppRootView()
+                .navigationDestination(for: RunnableEditor.self) { editor in
+                    EditorView(configuration: editor.configuration, dependencies: editor.dependencies)
```

Comment:
(nit) ideally, you'd want to show these as modals to match how it's used in the apps